libMBIN/Source/Models/Structs/GcSolarSystemData.cs

```diff
@@ -32,30 +31,42 @@ public class GcSolarSystemData : NMSTemplate
 
         /* 0x03B0 */ public Vector2f FreighterTimer;
         /* 0x03B8 */ public Vector2f PirateTimer;
-        /* 0x03C0 */ public Vector2f FlybyTimer;
+        /* 0x03C0 */  public Vector2f FlybyTimer;
         [NMS(Size = 8, Ignore = true)]
-        /* 0x03C8 */ public byte[] Padding3C8;
-        /* 0x0390 */ public GcSpaceStationSpawnData SpaceStationSpawn;
+        /* 0x03D0 */ public byte[] Padding3C8;
+        /* 0x0468 */ public GcSpaceStationSpawnData SpaceStationSpawn;
```

Comment:
The numbering here is messed up

Reply:
Padding3C8 should be at offset 0x3C8
The next field should be at 0x3D0

Reply:
But then there must be a padder after SpaceStationSpawn again because GcSpaceStationData is 120 in size?

Reply:
GcSpaceStationData is 0x140 in size

Reply:
I accidentally pressed start a review... by mistake... it's 0x140 in size, not 0x120..